src/app/home/home.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss'],
+})
+export class HomeComponent implements OnInit {
+  public form: FormGroup = new FormGroup({
+    title: new FormControl(),
+    description: new FormControl(),
+  });
+
+  public submit(): void {
+    this.form.markAllAsTouched();
```

Comment:
any specific reason for this?

Reply:
to ensure that all fields have been touched and properly filled out before submitting the form.